lblod/app-digitaal-loket
# Add open proces huis session role for all organizations

## Ticket Number

DL-5816

## Ticket Description

This PR adds the necessary code for the `Open Proces Huis` external module card, similar to what was done before for other external apps.

## How to Test

Check the corresponding [frontend PR](https://github.com/lblod/frontend-loket/pull/392).
